config.ru

```diff
@@ -4,4 +4,32 @@
 
 require_relative 'config/environment'
 
-run Rails.application
+ENV['PROMETHEUS_ENABLED'] ||= 'false'
+
+if ENV['PROMETHEUS_ENABLED'] == 'true'
```

Comment:
I'd prefer `ENV.fetch('PROMETHEUS_ENABLED', 'false')`. https://gist.github.com/stevenharman/5664318

Reply:
Will be changed.